### DISCUSSION TITLE
opciones de pago

### DISCUSSION THREAD

**Discussion by nataliacerron** [BlackBoxVision/react-native-mercadopago-px#90] [General]
Buenas, en la aplciación no se muestra el pago en efvo, quitaron esta opcion?

**Comment by JonatanSalas** [comment]
@nataliacerron no, eso vos lo configuras por medio de la preferencia de pago, no tiene nada que ver con el checkout, el checkout solo hace un fetch de la preferencia y dibuja la UI en funcion de esa configuracion.

Revisa tu backend porque ahi puede que este el problema.
